app/views/books/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= link_to("New", new_book_path, class: "btn btn-success mb-3 ml-3") %>
+<%= link_to("Users", "/users/", class: "btn btn-primary mb-3 ml-2") %>
```

Comment:
Instead of passing the "/users/" path directly we should use Rails helpers.

`rails routes | grep users`

which should show you this can be `users_path`.

We do this for a few reasons:
- Path helpers allows us to pass in params easily (like `format: :csv`)
- Path helpers will raise an exception if they no longer point to a correct path (EG: if you renamed the users routes to admins, this link would continue to render with a now invalid path when just the string is passed in. If you had `users_path` it would correctly raise an exception as no such path exists)
- Path helpers are easy to grep for rather than some variation of the users path as a string.

Reply:
On a separate note, and something we should pair on in the future when I'm not holidaying - what you have here for a menu is a good start and works now, but in future will become a headache for maintenance.

As you add more menu options with the current implementation it looks like you'd need to update several views to keep them consistent.

What we should try and do is move the navigation options in to the layout partial so they only have to be declared once.